src/types/utils.cpp

```diff
@@ -647,8 +647,23 @@ std::tuple<std::wstring, std::wstring> Utils::MangleStartingDirectoryForWSL(std:
                     // Tilde followed by non-space should be okay (like, wsl -d Debian ~/blah.sh)
                 }
 
+                // GH#11994 - If the path starts with //wsl$, then the user is

```

Comment:
careful -- there's also a `//wsl.localhost` version. we may want `//wsl` plus either a `$` or a `.`

Reply:
frick good point.